util.c

```diff
@@ -201,3 +201,60 @@ int r_mkdir(char *path)
 	}
 	return 0;
 }
+
+spawn_t spawn(const char *cmd, char *const argv[], unsigned int flags)
+{
+	spawn_t ret = { -1, -1, -1 };
```

Comment:
Use a better var name. Normally in a system call heavy function like this, ret holds the return value from a system call

Reply:
Suggestions welcome.

Reply:
proc_info? proc_fds? spawn_info?